src/Shell/Task/MigrationSnapshotTask.php

```diff
@@ -79,7 +79,7 @@ public function templateData()
             'collection' => $collection,
             'tables' => $tables,
             'action' => 'create_table',
-            'name' => $this->BakeTemplate->viewVars['name'],
+            'name' => $this->Template->viewVars['name'],
```

Comment:
The old way `BakeTemplate` was giving me undefined object errors and leaving me with a snapshot with no class name. Bake plugin seems to have changed to use `Template`.